oeps/oep-0021-proc-deprecation.rst

```diff
@@ -80,16 +80,18 @@ Process States
    :align: center
    :alt: A diagram that shows the state flow transitions. The process starts in
     the Proposed state and goes through the *Communicated*, *Accepted*,
-    *Deprecated*, *Removing*, and *Removed* states. If the proposal isn't 
+    *Deprecated*, *Removing*, and *Removed* states. If the proposal isn't
     *Accepted*, the state transitions directly from *Proposed* to *Abandoned*.
 
 As shown in the diagram above, the code removal process flows through the
 following states.
 
 *  Proposed_ - The starting state when removal is considered, analyzed, and
    documented.
-*  Communicated_ - The proposal is announced and its corresponding JIRA
-   ticket is updated with community feedback until the target Accepted date.
+*  Communicated_ - The proposal is announced to the general Open edX community
+   via the broadcast mechanisms defined below.  Specific notifications are
+   sent to all identified stakeholders.  The corresponding JIRA ticket
```

Comment:
_> Specific notifications are sent to all identified stakeholders._

Can we add a clause to the effect of "as best as possible"?  The real ideal solution for identifying and contacting all consumers of an API - is via our planned effort on Consumer-based Contracts.  Once we have consumer-based contract testing along with information of each consumer (internal and external), it will make identifying and contacting reliant consumers much more reliable.

In the meantime, a codebase search will have to do - but it won't catch everything - especially if consumers are outside of the `edx` Github organization or construct the API call in a way not anticipated by the deprecator.